spec/support/oidc_auth_helper.rb

```diff
@@ -98,6 +105,29 @@ def ial2_params(prompt: nil,
     ial2_params
   end
 
+  def vtr_params(
+    vtr:,
+    prompt: nil,
+    state: SecureRandom.hex,
+    nonce: SecureRandom.hex,
+    client_id: OIDC_ISSUER,
+    scope: 'openid email profile:name social_security_number',
+    tid: nil
+  )
+    vtr_params = {
+      client_id: client_id,
+      response_type: 'code',
+      vtr: Array(vtr).to_json,
+      scope: scope,
+      redirect_uri: 'http://localhost:7654/auth/result',
+      state: state,
+      nonce: nonce,
+    }
+    vtr_params[:tid] = tid if tid
```

Comment:
what's `tid`? is that the ID we added for inherited proofing?

Reply:
ooo nvm it was for attempts API, we can probably drop that here

Reply:
Here is a change to remove `tid` along with some explanation: https://github.com/18F/identity-idp/pull/10177